### ISSUE TITLE
Encode SAR (aspect ratio) into the bitstream

### ISSUE THREAD

**Issue by tdaede** [xiph/daala#21]
Some sources have nonsquare pixels. This is specified by the 'A' parameter in a y4m file. This should be put into daala_info and encoded/decoded so that the final output y4m has the same SAR encoded in it.

**Comment by tterribe** [comment]
There are already fields for this in daala_info, and it is read by encoder_example, stored in the stream, and output in dump_video. What are you doing to cause this to not be preserved?

**Comment by tdaede** [comment]
I think the bug report was on IRC using player_example and I didn't bother to actually check.

**Comment by tterribe** [comment]
Yes, player_example completely ignores these fields, to the best of my knowledge.

**Comment by tdaede** [comment]
Reopening this bug to track player_example.
